WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -15805,6 +15828,7 @@ e_armenia = {
 				OR = {
 					ai = no
 					religion_group = christian
+					culture = armenian
```

Comment:
I changed my mind on opposing all the de jure requirements in the  changes I made, and will instead discuss means to better the requirements, such as allowing all Armenians to create their culture's empire.